_rules/role-attribute-valid-value-674b10.md

```diff
@@ -64,9 +53,11 @@ Older browsers do not support more than one token in the value for a role attrib
 
 The `role` attribute is a set of [space separated tokens][]. Having a [whitespace](#whitespace) separated list of more than one token in the value of the role attribute is used for what is known as _fallback roles_. If the first token is not accessibility supported (or valid), the next one will be used for determining the [semantic role](#semantic-role) of the element, and so forth. Having the rule target attributes containing at least one non-[ASCII whitespace][] character ensures that there is at least one token in the set.
 
-Further reading:
+If an incorrect `role` attribute is specified on an element which is not [visible][], this is still likely to cause accessibility issues. However, this is not relevant to [Success Criterion 1.3.1 Info and Relationship][sc131] since nothing is conveyed through the presentation of that element.
```

Comment:
1.3.1 also targets "information and relationships that are implied by [...] auditory formatting". So, an element that is not visible can still be presented.

Reply:
That's truth, a whistling sound for the start of the main section and the footer section would need to be replicated programmatically.